pithos/pandora/pandora.py

```diff
@@ -161,7 +161,7 @@ def json_call(self, method, args=None, https=False, blowfish=True):
                 raise PandoraError("Login Error", code, submsg="Invalid username or password")
             elif code == ApiError.LISTENER_NOT_AUTHORIZED:
                 raise PandoraError("Pandora Error", code,
-                    submsg="A Pandora One account is required to access this feature. Uncheck 'Pandora One' in Settings.")
+                    submsg="A Pandora Plus account is required to access this feature. Uncheck 'Pandora Plus' in Settings.")
```

Comment:
Do you want me to changing the style of that one raise to shut flake8 up. It would look out of place. If you can ignore flake8 and merge this I can do a style clean up on pandora.py next?